src/styles/blocks/detail.scss

```diff
@@ -0,0 +1,53 @@
+.detail {
+  &__photo-container {
+    position: relative;
+
+    box-sizing: border-box;
+    width: 100%;
+    max-height: 130px;
+    overflow: hidden;
+    z-index: 0;
+    transition: transform 0.3s, z-index 0.3s;
+
+    &:hover {
+      transform: scale(105%);
+      z-index: 1;
```

Comment:
I think you don't need z-index here too

Reply:
> I think you don't need z-index here too

it should be, because if I remove it a picture that is next in the flow will go up the previous one, it looks ugly